### ISSUE TITLE
Add support for `info/collections`

### ISSUE THREAD

**Issue by michielbdejong** [mozilla-services/syncto#71]
SyncClient exposes various methods including [`info_collections`](https://github.com/mozilla-services/syncclient/blob/master/syncclient/client.py#L126), [`get_records`](https://github.com/mozilla-services/syncclient/blob/master/syncclient/client.py#L168) and [`get_record`](https://github.com/mozilla-services/syncclient/blob/master/syncclient/client.py#L223). Of these, Syncto only exposes `get_records` (from [here](https://github.com/mozilla-services/syncto/blob/master/syncto/views/collection.py#L64)) and `get_record` (from [here](https://github.com/mozilla-services/syncto/blob/master/syncto/views/collection.py#L64)).

It turns out that although `meta/global` lives on /storage/meta/global and can be fetched using either `get_records(meta)` or `get_record(meta, global)`, `info/collections` lives outside /storage, directly on /info/collections, and therefore we cannot access it via Syncto.

**Comment by Natim** [comment]
What do you need from info/collections ?
Is this related to: https://github.com/mozilla-services/syncto/issues/51 ?

**Comment by michielbdejong** [comment]
It's because of https://github.com/mozilla-services/docs/blob/368cc63d1e01583abef022ee5fc9b630c53eb102/source/sync/lifeofasync.rst#fetch-infocollections. See also https://github.com/mozilla-services/docs/issues/55

**Comment by michielbdejong** [comment]
I'm not so sure if we really need this though, or we can do without.

**Comment by michielbdejong** [comment]
It was for https://bugzilla.mozilla.org/show_bug.cgi?id=1204830#c0 but I guess it's just a 'nice to have' that can save some roundtrips if people sync 2 or more collections that don't change often.

**Comment by Natim** [comment]
Yes we can probably add an utility endpoint to provide this information if you need it.
